js/util/wizard_agency_search.js

```diff
@@ -132,11 +135,13 @@ function searchMatchingAgency(query, flatList, debug = false) {
     .map((word) => word.toUpperCase())
     .forEach((word) => {
       indexItems.forEach((item) => {
-        if (word === item.abbr || word === `US${item.abbr}`) {
-          item.score += word.length;
-          item.wordsMatched += 1;
-          matchedAbbr = true;
-          log(`Added ${word.length}pts to ${item.item.title} because user's query had word "${word}" matching item.abbr`, item);
+        if (!ignoreAbbrWords.includes(word)) {
+          if (word === item.abbr || word === `US${item.abbr}`) {
+            item.score += word.length;
+            item.wordsMatched += 1;
+            matchedAbbr = true;
+            log(`Added ${word.length}pts to ${item.item.title} because user's query had word "${word}" matching item.abbr`, item);
+          }
```

Comment:
Seems wise to be able to debug this behavior.

```suggestion
        if (ignoreAbbrWords.includes(word)) {
          log(`Word "${word}" was not considered for agency match because it is not an abbreviation.`);
        } else {
          if (word === item.abbr || word === `US${item.abbr}`) {
            item.score += word.length;
            item.wordsMatched += 1;
            matchedAbbr = true;
            log(`Added ${word.length}pts to ${item.item.title} because user's query had word "${word}" matching item.abbr`, item);
          }
```